src/platform/monitoring/frontend-metrics/feature-flag.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Returns whether the current environment should have metrics enabled.
+ * @returns {boolean}
+ * @module platform/monitoring/frontend-metrics/feature-flag
+ */
+export default function isMetricsEnabled() {
+  const environments = [
+    'preview'
+  ];
+
+  return !!environments.includes(__BUILDTYPE__);
```

Comment:
The `preview` environment is considered a Production environment, so I think this should be `vagovstaging`, but I'm not sure what the launch plan is. It may also be worth adding dev, `vagovdev`.

Is there any plan to incorporate this into Vets.gov prior to launching VA.gov?

Reply:
Reviewing the original [discovery](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/13291#issue-360974902) - looks like just dev and staging for VA.gov, and Vets.gov later.